tests/terraform/runner/extra_tf_plan_checks/security_group_rule_protocol_changed.py

```diff
@@ -0,0 +1,27 @@
+from __future__ import annotations
+
+from typing import Any
+
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+from checkov.terraform.plan_parser import TF_PLAN_RESOURCE_CHANGE_ACTIONS, TF_PLAN_RESOURCE_CHANGE_KEYS
+
+
+class SecurityGroupRuleProtocolChanged(BaseResourceCheck):
```

Comment:
can you also use this check in a test case to prove you have passing and failing checks?

Reply:
idk, can I? I am not an appdev so I did a lot of guessing/inferencing to arrive at what I just commited :sweat_smile: 

lmk if I did what I intended to